src/OrchardCore/OrchardCore.Cms/ServiceExtensions.cs

```diff
@@ -3,17 +3,18 @@
 using Orchard.Environment.Extensions;
 using Orchard.Environment.Extensions.Manifests;
 using Orchard.Environment.Shell.Data;
+using System.Linq;
 
 namespace Microsoft.Extensions.DependencyInjection
 {
     public static class ServiceExtensions
     {
-        public static IServiceCollection AddOrchardCms(this IServiceCollection services)
+        public static IServiceCollection AddOrchardCms(this IServiceCollection services, params string[] extraDefaultFeatureIds)
```

Comment:
This allows to set featureIds to be loaded on startup of the root project.
For example if you provide an alternate Setup shape in your SafeMode theme that replaces default shape you should initialize Orchard calling `services.AddOrchardCms("SafeMode");` on Startup